extension-requests/style.css

```diff
@@ -187,42 +192,53 @@ body {
 
 .extension-requests {
   display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+  grid-template-columns: repeat(auto-fill, minmax(200px, 22rem));
+  grid-auto-rows: 26rem;
   flex-wrap: wrap;
+  justify-content: center;
+  gap: 20px;
 }
 
 .extension-request {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  overflow: auto;
   text-align: left;
   border: 2px solid var(--black-transparent);
   padding: 1.25rem;
```

Comment:
also try using even measurements which helps in proper debugging and makes code look good

Reply:
Im not sure what you are referring to here. Can you please elaborate more.